src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/react-native-installation.mdx

```diff
@@ -0,0 +1,312 @@
+---
+title: React Native agent
```

Comment:
I'm thinking maybe a more descriptive title with word 'install' in it, maybe something like they did for this one https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation. could even do something like 'Install and use the React Native agent'

Reply:
I've been leaning towards things like "Monitor your React Native application" or something like that.

Reply:
I like the idea of "monitoring" and I also like the idea of being consistent as much as possible!